src/Repository/Projection/ManagedUserRepository.php

```diff
@@ -182,12 +183,32 @@ private function createFilterQueryBuilder(ManagedUsersFilter $filter): QueryBuil
             $typeExpression->add($and);
         }
 
-        if ($filter->includeCitizenProjectHosts()) {
+        if (true === $filter->includeCitizenProjectHosts()) {
             $typeExpression->add('json_length(u.citizenProjectsOrganizer) > 0');
         }
 
         $qb->andWhere($typeExpression);
 
+        // excludes
+        $excludes = $qb->expr()->andX();
+        if (false === $filter->includeCommitteeHosts()) {
+            $excludes->add('u.type = :type_h AND u.isCommitteeHost = 0');
+            $qb->setParameter('type_h', ManagedUser::TYPE_ADHERENT);
+        }
+
+        if (false === $filter->includeCommitteeSupervisors()) {
+            $and = new Andx();
+            $and->add('u.type = :type_s AND u.isCommitteeSupervisor = 0');
```

Comment:
could you remove all check for `u.type =` pls? as we have only one type here - `adherent`